specifyweb/frontend/js_src/lib/components/QueryBuilder/Results.tsx

```diff
@@ -256,14 +256,16 @@ export function QueryResults(props: QueryResultsProps): JSX.Element {
                 canFetchMore && !isFetching ? handleFetchMore : undefined
               }
             />
-            <QueryToForms
-              results={results}
-              selectedRows={selectedRows}
-              table={table}
-              totalCount={totalCount}
-              onDelete={handleDelete}
-              onFetchMore={isFetching ? undefined : handleFetchMore}
-            />
+            {isDistinct ? null : (
```

Comment:
(sad face)
okay.
if distict queries won't support QueryToForms, do we also need to disable other buttons for distinct queries (query to map, and etc...?)

Reply:
for now yes, same behavior as before

Reply:
For now just this change is sufficient